website/content/maintainers/community/contributor-growth-framework/incentivizing-non-code.md

```diff
@@ -3,7 +3,6 @@ title: "Incentivizing non-code contributions"
 linkTitle: "Non-Code Contributors"
 date: 2021-04-8
```

Comment:
Does testing/filing issues fall under non-code contributions? If it does, it would be worth adding it to this doc